bin/validate-payload-examples.js

```diff
@@ -6,26 +6,29 @@ const { ajv, validate } = require("../payload-schemas");
 let hasErrors = false;
 const payloads = `./payload-examples/api.github.com`;
 
-fs.readdirSync(payloads)
-  .filter((filename) => filename.endsWith(".json"))
-  .forEach((filename) => {
-    const [event] = filename.split(".");
-    const file = require(`../${payloads}/${filename}`);
+fs.readdirSync(payloads).forEach((event) => {
+  fs.readdirSync(`${payloads}/${event}`)
+    .filter((filename) => filename.endsWith(".json"))
+    .forEach((filename) => {
+      const file = require(`../${payloads}/${event}/${filename}`);
 
-    try {
-      const validationResult = validate(event, file);
-      if (!validationResult) {
-        console.error(`❌ Payload '${filename}' does not match schema`);
-        console.error(ajv.errors);
+      try {
+        const validationResult = validate(event, file);
+        if (!validationResult) {
+          console.error(
+            `❌ Payload '${event}/${filename}' does not match schema`
+          );
+          console.error(ajv.errors);
+          hasErrors = true;
+        } else {
+          console.log(`✅ Payload '${event}/${filename}' matches schema`);
+        }
+      } catch (err) {
+        console.error(`Missing schema for event '${event}'`, err);
```

Comment:
I don't think this change would be the best way to log errors. See #250

Reply:
Whoops yeah forgot I did that 😅 

I was going to fix that once I'd finished this PR - what do you think should happen, log or throw?